src/components/icons.tsx

```diff
@@ -1,25 +1,3 @@
-import { Spinner, SpinnerProps } from '@terra-ui-packages/components';
-import _ from 'lodash/fp';
-import React, { CSSProperties, ReactNode } from 'react';
+// pass-thru to be removed later.
 
-export { icon } from '@terra-ui-packages/components';
-
-export const centeredSpinner = (props: SpinnerProps = {}): ReactNode => {
-  const { size = 48, ...rest } = props;
-  const mergedProps = _.merge(
-    {
-      size,
-      style: {
-        display: 'block',
-        position: 'sticky',
-        top: `calc(50% - ${size / 2}px)`,
-        bottom: `calc(50% - ${size / 2}px)`,
-        left: `calc(50% - ${size / 2}px)`,
-        right: `calc(50% - ${size / 2}px)`,
-      } as CSSProperties,
-    },
-    rest
-  );
-
-  return <Spinner {...mergedProps} />;
-};
+export { icon, CenteredSpinner as centeredSpinner } from '@terra-ui-packages/components';
```

Comment:
If we need to continue supporting the render function, suggest defining a render function wrapper around the component in this module:

```ts
export const centeredSpinner = () => {
  return h(CenteredSpinner);
};
```